### DISCUSSION TITLE
How to increase max token input

### DISCUSSION THREAD

**Discussion by NunkiF** [infiniflow/ragflow#3360] [Q&A] [answered]
I noticed that when chat with the assistant, the tokens of Prompt is always less than 8192. 

How can I increase the max input prompt tokens to expand content in {knowledge}？

Now many LLM context windows have far exceeded 8K.

Thank you! @KevinHuSh

**Comment by KevinHuSh** [comment]
The context length depends on the LLM you've chosen. Chose anther LLM that supports longer context.

**Reply by NunkiF** [reply]
> The context length depends on the LLM you've chosen. Chose anther LLM that supports longer context.

Thanks for your reply! 

I'm using a third-party forwarded LLM, which uses an **OpenAI-API-Compatible** interface. 

Is this the reason why I can only input 8K context?

**Reply by KevinHuSh** [reply, marked as answer]
I got it.
There should be a blank to fill the context length while adding LLM.
We gona refine it later.
